.github/workflows/conformance.yml

```diff
@@ -61,34 +60,26 @@ jobs:
           ref: "conformance/v0.5.0-skipped-tests"
           path: "gateway-api"
 
-      - name: Setup Kind cluster
-        if: matrix.cluster-type == 'kind'
-        uses: ./consul-api-gateway/.github/actions/setup-kind
-        with:
-          metallb-config-path: "./consul-api-gateway/internal/testing/conformance/metallb-config.yaml"
-
       - name: Generate Unique Cluster Name
-        if: matrix.cluster-type == 'eks'
         id: unique
         run:
-            echo "::set-output name=clustername::$( echo consul-api-gateway-test-${{ github.run_number }}-$RANDOM )"
+          echo "::set-output name=clustername::$( echo consul-api-gateway-test-${{ github.run_number }}-$RANDOM )"
 
-      - name: Configure AWS credentials
-        if: matrix.cluster-type == 'eks'
-        uses: aws-actions/configure-aws-credentials@v1
+      - name: Setup Kind cluster
+        if: matrix.cluster-type == 'kind'
+        uses: ./consul-api-gateway/.github/actions/setup-kind
         with:
-          aws-region: ${{ env.AWS_REGION }}
-          role-to-assume: "arn:aws:iam::${{ secrets.AWS_ACCOUNT_ID }}:role/cicd-consul-apigateway-acceptance-tests"
-          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
-          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
-          role-duration-seconds: 7200
+          cluster-name: ${{ steps.unique.outputs.clustername }}
```

Comment:
Just using the generated cluster name for kind and EKS to reduce conditional logic, no harm